app/src/main/java/com/willowtree/vocable/room/PresetPhrasesRepository.kt

```diff
@@ -0,0 +1,7 @@
+package com.willowtree.vocable.room
+
+// TODO: MPV #467- this will take over the responsibilities for storing and fetching preset
+//                 phrases from the [ILegacyCategoriesAndPhrasesRepository]
+interface PresetPhrasesRepository {
+    suspend fun addPhrase(phrase: PresetPhraseDto)
```

Comment:
This might not even need to be part of the public interface, seeing as users can't actually _add_ a preset phrase. We can merge it now, I'm not concerned, but wanted to point that out.

Reply:
Yea removed already in my current work in progress 😅  just felt weird having an empty interface